go/vt/vtgate/vindexes/vschema.go

```diff
@@ -272,10 +277,11 @@ func (ks *KeyspaceSchema) findTable(
 // MarshalJSON returns a JSON representation of KeyspaceSchema.
 func (ks *KeyspaceSchema) MarshalJSON() ([]byte, error) {
 	ksJ := ksJSON{
-		Sharded:        ks.Keyspace.Sharded,
-		Tables:         ks.Tables,
-		ForeignKeyMode: ks.ForeignKeyMode.String(),
-		Vindexes:       ks.Vindexes,
+		Sharded:         ks.Keyspace.Sharded,
+		Tables:          ks.Tables,
+		ForeignKeyMode:  ks.ForeignKeyMode.String(),
+		MultiTenantSpec: ks.MultiTenantSpec,
```

Comment:
I don't think the order matters here, but will be nice to move this to the end for readability. That will be consistent with all other places.